test/functional/p2p_feefilter.py

```diff
@@ -48,6 +37,12 @@ def on_inv(self, message):
             if (i.type == MSG_TX):
                 self.txinvs.append(hashToHex(i.hash))
 
+    def wait_for_invs(self, test_framework, invs_expected):
+        def invs_match():
+            return sorted(invs_expected) == sorted(self.txinvs)
```

Comment:
could inline this with a lambda?

Reply:
Before I had done it using a lambda and an assignment, the the following way:
```
invs_match = lambda: sorted(invs_expected) == sorted(self.txinvs)
test_framework.wait_until(invs_match, timeout=60, lock=mininode_lock)
```
but then flake8 would give the following error:
`p2p_feefilter.py:41:9: E731 do not assign a lambda expression, use a def`

Directly using the lambda on place of the parameter of `wait_until` works though, done.